posts/announcing-first-cohort-serverless-champions-2017.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "The Serverless Champions Program HAS BEGUN!"
+description: "Do you have what it takes to be a Serverless Champion?"
+date: 2017-12-13
+layout: Post
+thumbnail: ''
+authors:
+  - RupakGanguly
+---
+
+# #Serverless4Life
+
+We care a lot about open source. Like, a _lot_. You all in the serverless community are the best we've seen. Period.
+
+And some of you? Blow our freaking minds with your unrelenting commitment to the future of computing.
+
+It was on a day not so long ago, as we were sweeping our mind-fragments off the floor for the dozenth time, in awe at your majesty, that we realized what we had to do. "These people deserve an award!" we said. Just like that. *These people deserve an award.*
```

Comment:
"These people deserve an award!" we said. Just like that. *These people deserve an award.* -> "These people deserve a recognition!" we said. Just like that. *These people deserve an award.*